adobe_vipm/flows/sync.py

```diff
@@ -254,6 +248,34 @@ def sync_agreements_by_next_sync(mpt_client, dry_run):
         sync_agreement(mpt_client, agreement, dry_run)
 
 
+def sync_agreements_by_coterm_date(mpt_client: MPTClient, dry_run: bool):
+    """
+    Synchronizes agreements by their active subscriptions renewed yesterday.
+    """
+    today = (datetime.now().today()).strftime("%Y-%m-%d")
+    yesterday = (datetime.now().today() - timedelta(days=1)).strftime("%Y-%m-%d")
+    yesterday_year_later = (datetime.now() + timedelta(days=364)).strftime(
+        "%Y-%m-%d"
+    )  # TODO: consider leap year
+    rql_query = (
+        "eq(status,Active)&"
+        f"any(subscriptions,or(eq(commitmentDate,{yesterday}),eq(commitmentDate,{yesterday_year_later})))&"
```

Comment:
To simplify the search here, I would recommend to use agreement parameter with externalId cotermDate. That is the same as subscriptions commitmentDate

Reply:
Done